code/game/turfs/change_turf.dm

```diff
@@ -149,7 +149,7 @@ GLOBAL_LIST_INIT(blacklisted_automated_baseturfs, typecacheof(list(
 	if(SSmapping.z_level_to_plane_offset[z])
 		var/area/thisarea = get_area(W)
 		if(thisarea.lighting_effects)
-			W.add_overlay(thisarea.lighting_effects[SSmapping.z_level_to_plane_offset[z]])
+			W.add_overlay(thisarea.lighting_effects[SSmapping.z_level_to_plane_offset[z] + 1])
```

Comment:
Why the +1 here?

Reply:
lighting_effects indexes at 1. plane offsets index at 0. I footgun'd myself